docs/res/format.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
```

Comment:
I'd rename this script to `format_docs.py`. It'll be easier to search for in IDE